README.md

```diff
@@ -184,3 +191,85 @@ Required dependencies:
 _The following is to be done in the root directory_
 
 Hooks are installed with when running `make install_git_hooks`, installing both the python pre-commit hooks as well as a commit-msg hook by cocogitto
+
+## OpenShift information
+
+### General information
+
+- [The Private Cloud as a Service Platform Technical Documentation](https://docs.developer.gov.bc.ca/)
+
+### Deploying the project
+
+To deploy the project into a a new namespace or to deploy another instance of the project into an existing namespace GitHub Environments along with Helm and GitHub Actions is used. The following steps can be used to as a reference to deploy:
+
+1. Create a new environment on the GitHub repository, set any protection rules as necessary. The environment will be used to hold the secrets needed for GitHub Actions to be passed to Helm.
+2. Add the following secrets and fill as appropriate:
+
+   - AWS_CLAM_S3_BUCKET
+   - AWS_ROLE_ARN
+   - AWS_S3_BACKUPS_BUCKET
+   - AWS_S3_BUCKET
+   - AWS_S3_KEY
+   - AWS_S3_REGION
+   - AWS_S3_SECRET_KEY
+   - CERTBOT_EMAIL
+   - CERTBOT_SERVER
+   - CLIENT_SECRET
+     - SSO Client Secret
+   - NEXT_PUBLIC_GROWTHBOOK_API_KEY
+   - OPENSHIFT_APP_NAMESPACE
+   - OPENSHIFT_METABASE_NAMESPACE
+     - Used for NetworkPolicy
+   - OPENSHIFT_METABASE_PROD_NAMESPACE
+     - Used for NetworkPolicy
+   - OPENSHIFT_ROUTE
+   - OPENSHIFT_SECURE_ROUTE
+   - OPENSHIFT_TOKEN
+
+3. Create any updated values as needed for your new deployment under `helm/app`. For example, if you named your environment `foo` you will create a file named `values-foo.yaml`
+4. Add an extra step to `.github/workflows/deploy.yaml` with updated job and environment name.
+5. Run the action!
+
+Note: there might be additional modifications or steps required to suit your specific needs, You might need to create independent workflows or Helm charts.
+
+### Disaster recovery documentation
+
+TBD
+
+### Cronjobs
+
+The project consists of several OpenShift [CronJobs](https://docs.openshift.com/container-platform/4.12/rest_api/workloads_apis/cronjob-batch-v1.html) to automatically run the following tasks:
+
+#### Incremental backup
+
+Managed by the PostgresCluster Operator (CrunchyDB), performs an incremental database backup at every 4 hours, starting at 1:00AM Pacific Time.
+
+#### Full backup
+
+As above managed by CrunchyDB. Performs a full backup of the database everyday at 1:00AM Pacific Time.
+
+#### Certbot
+
+As the name implies; a job that uses certbot to keep the TLS certificate up to date. Runs everyday at 5:00PM Pacific Time.
+
+#### Receive applications
+
+Marks all applications for a specific intake as received on the database. Runs twice a day at 10:00 AM and 10:00PM Pacific time.
+
+ANYTHING ELSE HERE?
```

Comment:
I'd just add that receive applications changes all applications with status `submitted` to `received`.